beacon_node/beacon_chain/src/attestation_verification.rs

```diff
@@ -816,9 +816,9 @@ impl<'a, T: BeaconChainTypes> IndexedUnaggregatedAttestation<'a, T> {
         committees_per_slot: u64,
         subnet_id: Option<SubnetId>,
         chain: &BeaconChain<T>,
-    ) -> Result<(u64, SubnetId), Error> {
-        let expected_subnet_id = SubnetId::compute_subnet_for_attestation_data::<T::EthSpec>(
-            indexed_attestation.data(),
+    ) -> Result<(u64, SubnetId), Error> { 
+        let expected_subnet_id = SubnetId::compute_subnet_for_attestation::<T::EthSpec>(
+            &attestation,
             committees_per_slot,
```

Comment:
using the provided attestation, instead of the indexed_attestation object here. is that ok?

Reply:
Can't think of a reason to use indexed_attestation